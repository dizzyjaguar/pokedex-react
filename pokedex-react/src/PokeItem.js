import React, { Component } from 'react';

export default class PokeItem extends Component {
    render() {
      return (
        <li className='poke-item'>
            <h2>{this.props.pokeItem.pokemon}</h2>
            <img alt='' src={this.props.pokeItem.url_image}></img>     
        </li>
      );
    }
    }