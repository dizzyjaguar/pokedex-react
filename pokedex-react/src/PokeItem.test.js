import React from 'react';
import { render } from '@testing-library/react';
import PokeItem from './PokeItem.js';
import renderer from "react-test-renderer";

test('renders App.js correctly', () => {
  const tree = renderer
    .create(<PokeItem pokeItem={{pokemon: '', url_image: '', hp: '', type_1: '' }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
