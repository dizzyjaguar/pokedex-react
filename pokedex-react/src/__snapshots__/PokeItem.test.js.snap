// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<li
  className="poke-item"
>
  <h2
    className="poke-name"
  >
    
  </h2>
  <img
    alt=""
    src=""
  />
  <p>
     HP: 
    
     
  </p>
  <p>
     Element: 
    
     
  </p>
</li>
`;
